libs/community/langchain_community/llms/oci_data_science_model_deployment_endpoint.py

```diff
@@ -454,14 +454,14 @@ def _default_params(self) -> Dict[str, Any]:
             "temperature": self.temperature,
             "top_k": self.k,
             "top_p": self.p,
-            **self.model_kwargs,
         }
 
     @property
     def _identifying_params(self) -> Dict[str, Any]:
         """Get the identifying parameters."""
+        _model_kwargs = self.model_kwargs or {}
```

Comment:
when user doesn't specify any model_kwargs, it will be None. We handle here because we want to keep the model_kwargs during serialization/deserialization. In other use case, model_kwargs has no hurt to be None or empty.

Reply:
I mean, shouldn't we do this in validate_environment or init method?

Reply:
Yes, we can do it in validate_environment, but I think model_kwargs has no hurt to be None or empty during initialization. So I handle it when it is being used, i.e. serialization time. But if you have strong preference, I can update it later in the feature branch.